x-pack/plugins/cases/server/connectors/case/index.ts

```diff
@@ -57,71 +58,69 @@ async function executor(
     throw new Error(msg);
   }
 
-  const { actionId, params, services } = execOptions;
+  const { actionId, params } = execOptions;
   const { subAction, subActionParams } = params;
   let data: CaseExecutorResponse | null = null;
 
-  const { scopedClusterClient } = services;
-  const casesClient = await factory.create({
-    request: undefined,
-    savedObjectsService: undefined,
-    scopedClusterClient,
-  });
+  let casesClient: CasesClient | undefined;
 
   if (!supportedSubActions.includes(subAction)) {
     const errorMessage = `[Action][Case] subAction ${subAction} not implemented.`;
     logger.error(errorMessage);
     throw new Error(errorMessage);
   }
 
-  if (subAction === 'create') {
-    try {
-      data = await casesClient.cases.create({
-        ...(subActionParams as CasePostRequest),
-      });
-    } catch (error) {
-      throw createCaseError({
-        message: `Failed to create a case using connector: ${error}`,
-        error,
-        logger,
-      });
+  // When the actions framework provides the request and a way to retrieve the saved objects client with access to our
```

Comment:
@cnasikas let me know what you think of this. This could shouldn't ever be executed because the the `throw` above should happen anyway. Just makes typescript check happy.